dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -216,6 +216,43 @@ def __add__(self, other):
         merged_profile.sample_size = self.sample_size + other.sample_size
 
         return merged_profile
+    
+    def diff(self, other_profile, options=None):
+        """
+        Finds the differences for two unstructured text profiles
+
+        :param other_profile: profile to find the difference with
+        :type other_profile: TextProfiler
+        :param options: options for diff output
+        :type options: dict
+        :return: the difference between profiles
+        :rtype: dict
+        """
+        cls = self.__class__
+        if not isinstance(other_profile, cls):
+            raise TypeError("Unsupported operand type(s) for diff: '{}' "
+                            "and '{}'".format(cls.__name__,
+                                              other_profile.__class__.__name__))
+
+        diff = {}
+
+        diff["vocab"] = utils.find_diff_of_lists_and_sets(
+            list(self.vocab_count.keys()),
+            list(other_profile.vocab_count.keys()))
+
+        diff["vocab_count"] = utils.find_diff_of_dicts_with_diff_keys(
+            dict(self.vocab_count.most_common(self._top_k_chars)),
```

Comment:
I'd prefer using the whole set instead of selecting the top ones. Otherwise, it'd be not consistent with the diff['vocab']

Reply:
We dont keep track of the top k vocab so it cant be consistent